dvc/repo/experiments/__init__.py

```diff
@@ -398,7 +398,7 @@ def reproduce_one(
             self.reset_checkpoints()
 
         if not (queue or tmp_dir or machine):
-            staged, _, _ = self.scm.status()
+            staged, _, _ = self.scm.status(untracked_files="no")
             if staged:
```

Comment:
While having some discussions related to using pygit2 for `status` for performance reasons with @dtrifiro, I noticed that this is equivalent to `git diff --staged --quiet`. So, this is equivalent to following:
```suggestion
            if git.pygit2.repo.diff(cached=True):
```